elixir/lib/sqlite_vss.ex

```diff
@@ -0,0 +1,245 @@
+       :binary
+     )
+   ),
+   order_by: [asc: v0.distance],
+   limit: ^100,
+   select: [:rowid, :distance]),
+   on: v1.rowid == p0.id>
+
+  """
+  @spec search_query(Ecto.Query.t(), String.t(), String.t(), binary(), integer() | none()) :: Ecto.Query.t()
+  def search(query, table_name, embedding_column_name, vector, limit \\ 100) do
+    subquery =
+      from(s in table_name,
+        where:
+          fragment(
+            "vss_search(?, ?)",
+            field(s, ^String.to_existing_atom(embedding_column_name)),
+            type(^vector, :binary)
+          ),
+        select: [:rowid, :distance],
+        order_by: s.distance,
+        limit: ^limit
+      )
+
+    from(p in query,
+      join: s in subquery(subquery),
+      on: s.rowid == p.id
+    )
+  end
+
+  @doc false
+  def start(_, _) do
+    unless Application.get_env(:sqlite_vss, :version) do
+      Logger.warn("""
+      sqlite_vss version is not configured. Please set it in your config files:
+
+          config :sqlite_vss, :version, "#{latest_version()}"
+      """)
+    end
+
+    Supervisor.start_link([], strategy: :one_for_one)
+  end
+
+  @doc """
+  Returns the configured sqlite_vss version.
+  """
+  def configured_version do
+    Application.get_env(:sqlite_vss, :version, latest_version())
+  end
+
+  @doc """
+  Installs sqlite_vss with `configured_version/0`.
+  """
+  def install(base_url \\ default_base_url()) do
+    tmp_dir =
+      freshdir_p(Path.join(System.tmp_dir!(), "sqlite_vss")) ||
```

Comment:
any chance we can download straight to `bin_dir` and avoid a tmpdir? Then just delete the `.tar.gz` afterwards

Also heads up, i'll be bunding the two `.tar.gz` files for `vector0`/`vss0` into a single `tar.gz` probably this week, but I can make that change here when that happens

Reply:
Awesome, thanks for the heads up!  Was this a part of [v0.1.1-alpha.7](https://github.com/asg017/sqlite-vss/commit/7f9e7c32d58d41f5d3f62a8fa8bd2bd00f4693c5)?